pkg/encoding/utils/pointsIO.go

```diff
@@ -10,11 +11,27 @@ import (
 	bls "github.com/Layr-Labs/eigenda/pkg/kzg/bn254"
 )
 
+const (
+	// Num of bytes per G1 point.
+	G1PointBytes = 32
+	// Num of bytes per G2 point.
+	G2PointBytes = 64
+)
+
 type EncodeParams struct {
 	NumNodeE  uint64
 	ChunkLenE uint64
 }
 
+func ReadFile(reader *bufio.Reader, numBytesToRead uint64) ([]byte, error) {
+	buf := make([]byte, numBytesToRead)
```

Comment:
Would it be better if adding a check between the bytes requested and byte actually read?

Reply:
Yeah I was thinking that as well. But there is guarantee from API already: ReadFull() ensures that "On return, n == len(buf) if and only if err == nil" already: https://pkg.go.dev/io#ReadFull